sdk/keyvault/Azure.Security.KeyVault.Administration/tests/AccessControlTestBase.cs

```diff
@@ -26,7 +26,7 @@ public abstract class AccessControlTestBase : AdministrationTestBase
         internal string _objectId;
 
         public AccessControlTestBase(bool isAsync, KeyVaultAdministrationClientOptions.ServiceVersion serviceVersion, RecordedTestMode? mode)
-            : base(isAsync, serviceVersion, mode)
+            : base(isAsync, serviceVersion, mode, useLegacyTransport: true)
```

Comment:
Can you clarify why we can't use the proxy?

Reply:
There were a few HSM tests that I wasn't able to re-record. I was planning to dig into the failures after merging this.